src/main/java/org/apache/commons/scxml2/env/jexl/JexlEvaluator.java

```diff
@@ -185,7 +195,13 @@ protected JexlEngine createJexlEngine() {
         // See javadoc of org.apache.commons.jexl2.JexlEngine#setFunctions(Map<String,Object> funcs) for detail.
         final Map<String, Object> funcs = new HashMap<>();
         funcs.put(null, JexlBuiltin.class);
+
         JexlPermissions permissions = JexlPermissions.RESTRICTED.compose("org.apache.commons.scxml2.*");
+
+        if(customAllowedClasses != null && customAllowedClasses.length > 0) {
```

Comment:
Fix formatting.

Reply:
Is done. Variable is rename to `customJexlPermissions` now ([see](https://github.com/apache/commons-scxml/pull/154/files#diff-e9aaca46c5c2de0f4a9fde75d3e1bbd108c244261ab9019693dda200ea4b0846R201]))